### PR TITLE
removed old codebase of certificates

### LINKED ISSUE #1958: Move Certificates as extension
**Description**

We replace our current implementation with the extensibility option.

**Acceptance Criteria**

- [ ] Validate with Product Owner if the current implementation is correct
- [ ] Replace the current implementation with extensibility
- [ ] Deploy to stage and Test all the edge cases on stage
- [ ] Deploy to production
- [ ] Remove old codebase
- [ ] Adjust Tests to use extensions

### PR DESCRIPTION
**Pull request by chriskari**
**Description**

Changes proposed in this pull request:

- removed old codebase, extension and tests

**Related issue(s)**
closes #1958

### CODE REVIEW

File: public/i18n/en.yaml [hunk 1]
```diff
@@ -58,48 +58,6 @@ btp-service-bindings:
...
-    csr: Certificate Signing Request is an alternative way to provide common name, DNSNames, and other information.
-    dns-names: DNS Names are the optional additional domain names of the certificate.
-    existing-secret: Use an existing Secret for storing the certificate if enabled. Otherwise a new Secret will be created.
-    expired: Certificate expired
-    issuer: Certificate issuer. If no issuer is given, the default issuer is used.
-    ready: Certificate issued
-    renew: Trigger a certificate renewal if enabled.
-    secret-name: Name of the Secret object to use for storing the certificate.
-    secret-ref: Reference of the existing Secret object to use for storing the certificate.
-    will-expire: Certificate will expire in {{daysToExpire}} {{daysForm}}.
```
Comment (Lyczeq) [inline public/i18n/en.yaml:100]: You deleted some of the important translations there. The component `CertificateDate` still uses them. They're quite related to Secrets, so you can move them right there.

**Comment by Lyczeq** [comment]
blocked by https://github.com/kyma-project/busola/pull/2308 and https://github.com/kyma-project/busola/pull/2309

File: tests/integration/tests/namespace/test-custom-resources.spec.js [hunk 2]
```diff
@@ -29,13 +27,7 @@ context('Test Custom Resources', () => {
       .contains('Certificates')
       .click();
 
-    cy.get('[aria-label="title"]')
-      .contains('Certificates')
-      .should('be.visible');
-
-    cy.contains('cypress-test-name').should('be.visible');
-
-    cy.contains('certificates.cert.gardener.cloud').click();
```
Comment (mrCherry97) [inline tests/integration/tests/namespace/test-custom-resources.spec.js:38]: why it's removed?

**Review by mrCherry97 — CHANGES_REQUESTED** [review]
`src/components/Predefined/Details/Certificate/CertificateRefs.scss` also could be removed, it's not used anywhere

**Review by mrCherry97 — DISMISSED** [review]

**Review by NHingerl — APPROVED** [review]

**Review by mrCherry97 — APPROVED** [review]
